moped-editor/src/views/projects/projectView/ProjectStatusBadge.js

```diff
@@ -160,9 +160,7 @@ const useChipStyles = makeStyles((theme) => ({
     backgroundColor: ({ phaseKey }) =>
       getStyle(theme, phaseKey ?? "").background,
   },
-  leftMargin: {
-    // marginLeft: "1rem",
-  },
+  leftMargin: {},
```

Comment:
I was going to remove this style completely, but this got me thinking of where else the projectstatusbadge component is used (forgot to check before approving Frank's PR) and i discovered a regression https://github.com/cityofaustin/atd-data-tech/issues/16777